### PR TITLE
Fix stack overflow in LocalizableErrorArgument

### PR DESCRIPTION
**Pull request by agocke**
@jaredpar

### CODE REVIEW

File: src/Compilers/VisualBasic/Portable/Errors/LocalizableErrorArgument.vb [hunk 1]
```diff
@@ -29,7 +34,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         ''' <param name="format">A string to use for formatting.</param>
         ''' <param name="formatProvider">An object that supplies culture-specific format information about format.</param>
-        Public Function ToString_IFormattable(format As String, formatProvider As IFormatProvider) As String Implements IFormattable.ToString
+        Public Overloads Function ToString(format As String, formatProvider As IFormatProvider) As String Implements IFormattable.ToString
```
Comment (jaredpar) [inline src/Compilers/VisualBasic/Portable/Errors/LocalizableErrorArgument.vb]: Please keep the public API name.  If we feel strongly about it we should bring it up separately with compat council

Comment (agocke) [inline src/Compilers/VisualBasic/Portable/Errors/LocalizableErrorArgument.vb]: Can I add the ToString Overload at least for accessibility?

Comment (jaredpar) [inline src/Compilers/VisualBasic/Portable/Errors/LocalizableErrorArgument.vb]: Lets bring that up with compat council.  Would like to handle fixing this bug and the API implications separately.

**Comment by jaredpar** [comment]
:+1:

**Pull request merged**